Library/Formula/ddar.rb

```diff
@@ -9,7 +9,7 @@ class Ddar < Formula
 
   depends_on "xmltoman" => :build
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on "protobuf"
```

Comment:
The `ddar` command has a `#!/usr/bin/python` line, so it runs with the system python. On my machine where `ddar` works, the system python has `/usr/local/lib/python2.7/site-packages` and some other `/usr/local` and Cellar locations on its `site.path`. (`/usr/bin/python -c 'import sys; print "\n".join(sys.path)'`) On the system where `ddar` doesn't work, no `/usr/local` locations appear in `sys.path` for the system python.

I don't know what's causing the different paths on the two systems. On both, I have the Homebrew `python` installed, and it's first on my path.

Perhaps this is a side effect of having previously built some Homebrew python bindings against the system python in the past on one system but not the other?